File: src/Backend/LSWBackend/LSWBackend/Dtos/OfferDto.cs
namespace LSWBackend.Dtos;

public class OfferDto
{
    public string Title { get; set; } = null!;
    public string? Description { get; set; }
    public List<string> Teachers { get; set; } = null!;
    public double? Price { get; set; } = null!;
    public List<string> StartDates { get; set; } = null!;
    public List<string> EndDates { get; set; } = null!;
    public int CurrentCount { get; set; }
    public int MaxCount { get; set; }
    public int MinCount { get; set; }
    public string? Location { get; set; }
    public string? MeetingPoint { get; set; }
    public bool? Enrolled { get; set; }
}

File: src/Frontend/src/app/models/offerDto.ts
export interface OfferDto {
    title: string;
	description: string | null;
	teachers : string[];
	price : number | null;
	startDates : string[];
	endDates : string[];
    currentCount : number;
	maxCount : number;
	minCount : number;
	location : string | null;
	meetingPoint : string | null;
	enrolled : boolean | null;
}